packages/react-core/src/components/Icon/examples/IconProgress.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Icon, Spinner } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+export const IconProgress: React.FunctionComponent = () => (
+  <React.Fragment>
+    <Icon isInProgress progressIcon={<Spinner size="md" isSVG aria-label="Loading..." />} containerSize="md">
```

Comment:
I wonder if we should update these examples so that the progress state is triggered by clicking on something. And either clicking toggles it off, too, or there is some kind of timeout. One reason this component exists is to create a consistently sized box that a user can swap icons in and out of without the box resizing and shifting around any text beside it.

So maybe something like:

```
<label><checkbox onClick="toggles <icon>'s isInProgress prop"> toggle the icon's `isInProgress` state</label>

<Icon progressIcon={...} /> Some sort of descriptive thing the icon represents
```

Reply:
Agree - I put them side by side in the core example as comparison, but here we can allow them to switch between.